packages/eslint-plugin-svelte/src/rules/prefer-const.ts

```diff
@@ -0,0 +1,88 @@
+import { createRule } from '../utils';
+import type { TSESTree } from '@typescript-eslint/types';
+
+import { defineWrapperListener, getCoreRule } from '../utils/eslint-core';
```

Comment:
I changed this package to ESM.
https://github.com/sveltejs/eslint-plugin-svelte/pull/932

Could you please merge the main branch and fix it by adding `.js` to the import statement of this PR?